# this is a single apriltag detection
geometry_msgs/PoseStamped pose
uint32 id
string name